index.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
-var Requested = require('./requested')
-  , https = require('https')
-  , http = require('http');
+module.exports = (function () {
+  if ('XMLHttpRequest' in global) {
+    return require('./browser');
+  }
 
-module.exports = Requested.extend({
-  initialize: function initialize() {
+  var XMLHttpRequest = global.XMLHttpRequest = require('node-http-xhr');
```

Comment:
Introducing global variables in node environments is usually frowned upon, is there no way to bypass this?